azure-pipelines.yml

```diff
@@ -146,7 +155,12 @@ stages:
       - task: NuGetToolInstaller@1
         displayName: "Install NuGet"
 
-      - powershell: .\scripts\build\set-azp-project-version-variable.ps1
+      - download: current
+        artifact: Azp-Variables
+        displayName: "Download AZP Variables pipeline artifact"
+
+      - powershell: .\scripts\build\set-azp-variables.ps1
+        displayName: "Set AZP Variables"
```

Comment:
These 6 lines repeats 5 times in this file. Shall I extract it to `azp-variables-steps.yml` template? Usage would be single line step:
```
- template: azp-variables-steps.yml
```

Reply:
yes that would be nice